OSDOCS-6146: Fix verification steps and misc copy edits (#60414)
openshift/openshift-docs: modules/microshift-deploying-a-load-balancer.adoc

:_content-type: PROCEDURE
[id="microshift-deploying-a-load-balancer_{context}"]

> = Deploying a load balancer on a workload

Reviewer: I'm not sure "on" is the right word there. Maybe "for"?

+
[NOTE]
====
You must ensure that the `port` parameter is a host port that is not occupied by other `LoadBalancer` services or {product-title} components.
====

. Verify that the service file exists, that the external IP address is properly assigned, and that the external IP is identical to the node IP by running the following command:
+
[source,terminal]
----
$ oc get svc -n $NAMESPACE
----
+
.Example output
[source,terminal]
----
NAME    TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)        AGE
nginx   LoadBalancer   10.43.183.104   192.168.1.241   81:32434/TCP   2m
----

.Verification

> * The following command forms five connections to the example `nginx` application using the external IP address of the `LoadBalancer` service config, then lists those server IP addresses. Verify that the load balancer sends requests to all the running applications with the following command:

Reviewer 2: ```suggestion
* The following command forms five connections to the example `nginx` application using the external IP address of the `LoadBalancer` service config. The result of the command is a list of those server IP addresses. Verify that the load balancer sends requests to all the running applications with the following command:
```

Reviewer 2: I tried to break this up because it was a little confusing to me.

+
[source,terminal]
----
EXTERNAL_IP=192.168.1.241
seq 5 | xargs -Iz curl -s -I http://$EXTERNAL_IP:81 | grep X-Server-IP
----
+

> Your output must contain different IP addresses to show that the load balancer is successfully distributing the traffic to the applications.

Reviewer: "must" feels wrong. This step is trying to show the user how to verify that what they did in the earlier steps is working correctly. 

```suggestion
If the load balancer is successfully distributing the traffic to the applications, the output of the previous command will contain different IP addresses.
```
